src/applications/gi-sandbox/sass/gi.scss

```diff
@@ -31,6 +31,32 @@
       padding-right: 0.4em;
     }
   }
+  .use-my-location-container{
```

Comment:
If these classes are location specific they should go in `src/applications/gi-sandbox/sass/partials/_gi-search-location.scss`